yarn.lock

```diff
@@ -2726,9 +2726,9 @@ analytics-events@^1.2.0:
   integrity sha1-sDCnqCv+tF7w2qdNIM+ai6TUvcs=
 
 analytics-events@^2.0.2, analytics-events@^2.1.0:
-  version "2.2.0"
```

Comment:
@kdaswani is this change intentional? curious if this will impact other destinations as well.

Reply:
yes! I had to add a new event for `Video Playback Exited` to analytics-events so we needed to update the version here.